### DISCUSSION TITLE
if you guys are using ipam XtratusCloud provider then please if you can +1 the feature request

### DISCUSSION THREAD

**Discussion by omerzubair** [Azure/ipam#245] [Show and tell]
You guys might already be aware that the terraform provider is there for use in conjunction to ipam solution. at this link https://registry.terraform.io/providers/XtratusCloud/azureipam/latest/docs.
i can confirm the provider works however need the following feature. Please if you can +1 it if you think you will drive value out of it. :) 
Tagging @DCMattyG as well

**New feature request:** 
https://github.com/XtratusCloud/terraform-provider-azureipam/issues/11

**Comment by DCMattyG** [comment]
Thanks for posting this here @omerzubair. I upvoted the feature 👍
